### ISSUE TITLE
Remove title from search snippet results

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#1951]
This redundancy is unnecessary and rather ugly. Also, it prevents eventually showing the document contents matches.

![screenshot from 2018-10-03 20-01-40](https://user-images.githubusercontent.com/489831/46429634-33c5b300-c747-11e8-9a15-0506c8cee60e.png)

**Comment by RafaPolit** [comment]
@txau now, with the CSS escaping and also, as planned, we could eventually not show the title on a card (it is already being used in some instances), maybe we need to rethink this?

**Comment by txau** [comment]
@RafaPolit what is the CSS escaping?

**Comment by RafaPolit** [comment]
Yes, I should have been more explicit.  We now class-name cards and rows.  This allows for targeted template variations.  One of them is the chance to hide names on certain templates on cards. Like here: https://best-artworks.uwazi.io/

We also have talked frequently about allowing the title and date created properties to be 'normal' fields that allow you to not show in cards, or even sort them in different places of the card (not only at the top).

So, if someone goes for any of these scenarios, having the snippet show the source of the match will not necessarily be as redundant.  So, I do not think removing it at this point is the best approach.
